lib/chutney/TorNet.py

```diff
@@ -1324,10 +1356,10 @@ def getNodeDirInfoStatusPattern(self, dir_format):
             return r'^router ' + nickname + " "
         elif md:
             # Not yet implemented, see #33428
-            # r'^id ed25519 " + ed25519_identity (end of line)
+            return r'^id ed25519 ' + re.escape('ed25519_id')
```

Comment:
'ed25519_id' is a string, but you want the value of the variable ed25519_id:
```suggestion
            return r'^id ed25519 ' + re.escape(ed25519_id)
```